app/src/main/java/org/p2p/wallet/updates/SocketUpdatesManager.kt

```diff
@@ -37,7 +38,7 @@ private const val TAG = "Sockets:SocketUpdatesManager"
 class SocketUpdatesManager(
     appScope: AppScope,
     private val environmentManager: NetworkEnvironmentManager,
-    private val connectionStateProvider: NetworkConnectionStateProvider,
+    private val connectionStateProvider: ConnectionManager,
```

Comment:
ConnectionManager measures the internet state more accurately so i've changed it

Reply:
should we delete NetworkConnectionStateProvider?

Reply:
I assume yes, connectivity manager doesn't check the internet in many cases, sometimes even if we turned off all transports it still saying that everything okay. But let's keep it for a while until the testers have tested current bug.